gh-460: Data Elements of Data Types not displayed

jon-bowen opened the pull request.

pjmonks reviewed (changes requested):
The fix itself looks good and functions correctly 👍 

The only issue found during the build was a linting issue. Fortunately, if you run `npm run eslint` the error in question will automatically fix itself because it's so simple. If you could push up that change I'll be happy to approve.

jon-bowen commented:
Fixed whitespace issue.
I also added an npm script "eslint-nofix-quiet" so I can see what errors there are without being wiped out with the 811 warnings.
